src/parser.rs

```diff
@@ -60,8 +60,6 @@ pub enum ParseError {
     InvalidRegex(String, Location),
     #[error("Expected integer constant in regex capture at {0}")]
     InvalidRegexCapture(Location),
-    // TODO: The positions in the wrapped QueryError will be incorrect, since they will count the
-    // row/column from the start of the query, not from the start of the file.
```

Comment:
This is removed but I don't a corresponding code change. Was it fixed earlier?

Reply:
Yes, it was fixed in an [earlier commit](https://github.com/tree-sitter/tree-sitter-graph/commit/f80d5a2cba316a09aa08425a78a3e08d4d645817) but I overlooked this comment then.